exec/java-exec/src/main/java/org/apache/drill/exec/planner/FileSystemPartitionDescriptor.java

```diff
@@ -212,8 +223,12 @@ public TableScan createTableScan(List<PartitionLocation> newPartitionLocation) t
     }
 
     if (scanRel instanceof DrillScanRel) {
-      final FileSelection newFileSelection = new FileSelection(null, newFiles, getBaseTableLocation());
-      final FileGroupScan newGroupScan = ((FileGroupScan)((DrillScanRel)scanRel).getGroupScan()).clone(newFileSelection);
+//      final FormatSelection formatSelection = (FormatSelection)((DynamicDrillTable)scanRel.getTable()).getSelection();
+      final FormatSelection formatSelection = (FormatSelection)table.getSelection();
+      final FileSelection newFileSelection = new FileSelection(null, newFiles, getBaseTableLocation(),
+          cacheFileRoot, formatSelection.getSelection().getDirStatus());
+      final FileGroupScan newGroupScan =
+          ((FileGroupScan)((DrillScanRel)scanRel).getGroupScan()).clone(newFileSelection, cacheFileRoot);
```

Comment:
I agree it seems redundant second parameter.  I don't recall why I needed it at some point  but let me see if I can get rid of it.